lib/utils.js

```diff
@@ -384,13 +384,7 @@ function makeParentFunctionAsync(j, path) {
  */
 function dropAndThen(j, root) {
   let replacements = root
-    .find(j.CallExpression)
-    .filter(({ node }) => {
-      return j.Identifier.check(node.callee)
-        && node.callee.name === 'andThen'
-        && node.arguments.length === 1
-        && j.FunctionExpression.check(node.arguments[0])
-    })
+    .find(j.CallExpression, { callee: { name: 'andThen' } })
```

Comment:
Yeah, that's nice. I only learned about this API when working on ember-modules-codemod, back when I wrote this I was not aware this is possible. But much nicer! Many places in this codebase can probably be simplified like this. I miss some good (and complete) documentation on jscodeshift... 😩

Reply:
Totally agreed! The docs are... lacking to say the least.